modules/redis/variables.tf

```diff
@@ -72,9 +72,9 @@ variable "memory_size_gb" {
 }
 
 variable "redis_version" {
-  description = "The redis version in the form REDIS_4_0."
+  description = "The redis version in the form REDIS_7_0."
   type        = string
-  default     = "REDIS_4_0"
+  default     = "REDIS_7_0"
```

Comment:
Når vi endrer denne tror jeg vi må legge på en 
lifecycle {ignore_changes} på redis_version her: https://github.com/entur/terraform-google-memorystore/pull/35/files?diff=split&w=0#diff-ae26dcd467e1403a2b5dad2c13ea40d1aa0b80f4ea9a7f07a106d51fbd55179eL17
Ellers vil vel eksisterende redis bli oppgradert/destroyed..

Hvis ikke må vi kommunisere det veldig bra som @AlexanderBrevig sier :-)

Reply:
Det er muligens at dette uansett blir et problem ved enabling av auth, da[ memorystore dokumentasjonen](https://cloud.google.com/memorystore/docs/redis/manage-redis-auth) til Google ikke presiserer om dette medfører en restart av instansen.

Oppgradering til den nye modulen vil uansett medføre noe nedetid, da connections i gamle pods til redis ikke vil fungere i perioden det skjer en ny rollout deployment som med støtte for auth.

Reply:
Konklusjon etter intern platform diskusjon:
Redis versjon forandres til 7_0, og auth er disabled by default.